integrations/issue_test.go

```diff
@@ -107,7 +107,7 @@ func TestNoLoginViewIssue(t *testing.T) {
 	MakeRequest(t, req, http.StatusOK)
 }
 
-func testNewIssue(t *testing.T, session *TestSession, user, repo, title string) {
+func testNewIssue(t *testing.T, session *TestSession, user, repo, title, content string) string {
```

Comment:
nit: from the signature, it's not obvious what the function returns. Could we add a brief comment specifying that the created issue's URL is returned?